platforms/web/.env

```diff
@@ -1,4 +1,4 @@
-APP_API_BASE_URL=https://cdn.jwplayer.com
+APP_API_BASE_URL=https://cdn-dev.jwplayer.com
```

Comment:
Prob shouldn't commit this change to the main .env file. You can use .env.local or .env.dev.

Reply:
This PR should remain draft, we need this to look into dev, as the middleware at the moment communicates only with dev. We don't have all the services ready in other envs.